schema/crdb/dbinit.sql

```diff
@@ -2513,20 +2514,40 @@ CREATE TABLE omicron.public.switch_port_settings_address_config (
 /*******************************************************************/
 
 /*
- * Metadata for the schema itself.  This version number isn't great, as there's
+ * Metadata for the schema itself. This version number isn't great, as there's
  * nothing to ensure it gets bumped when it should be, but it's a start.
```

Comment:
What's non-great about the version number?

Reply:
Dunno, this was a comment from before my PR. Do you want me to remove it? It's perhaps not sufficient enough (now) to identify any other information about "how to apply the change".